### PR TITLE
CHUI-50: Add country data for BRA, ARG, BOL and USA

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?

This PR adds the configuration apps for Brazil, Bolívia, Argentina and United States. This depends on vtex-apps/country-data-settings#1.

#### How should this be manually tested?

[Workspace](https://steps--checkoutio.myvtex.com/checkout/#/profile). Follow the same test plan as the linked PR above.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: packages/bol/vtex.country-data-settings/configuration.json [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+    },
+    "street": {
+      "label": "street",
+      "required": true
+    },
+    "number": {
+      "maxLength": 750,
+      "label": "number",
+      "required": true,
+      "autoComplete": "nope"
```
Comment (lucasecdb) [inline packages/bol/vtex.country-data-settings/configuration.json]: this doesn't look right.. 🤔

File: packages/bra/vtex.country-data-settings/configuration.json [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    },
+    "street": {
+      "label": "streetRoadAvenue",
+      "required": true
+    },
+    "number": {
+      "label": "numberOption",
+      "maxLength": 750,
+      "required": true,
+      "autoComplete": "nope"
```
Comment (lucasecdb) [inline packages/bra/vtex.country-data-settings/configuration.json]: same

File: packages/usa/vtex.country-data-settings/configuration.json [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+    },
+    "street": {
+      "label": "addressLine1",
+      "required": true
+    },
+    "number": {
+      "label": "number",
+      "maxLength": 750,
+      "hidden": true,
+      "autoComplete": "nope"
```
Comment (lucasecdb) [inline packages/usa/vtex.country-data-settings/configuration.json]: same

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
 # Countries Data
+
+Monorepo containing the configuration apps for each country. These apps configures
```
Comment (marcoskwkm) [inline README.md]:
```suggestion
Monorepo containing the configuration apps for each country. These apps configure
```

File: README.md [hunk 5]
```diff
@@ -1 +1,22 @@
...
+the [`vtex.country-data-settings`](https://github.com/vtex-apps/country-data-settings) service app,
+which is currently being used to power Checkout's profile and shipping forms.
+
+## Adding a new country
+
+To add a new country configuration app, use the script `./scripts/create-country-app.js`. The syntax
+supported by this is script is the following:
```
Comment (marcoskwkm) [inline README.md]:
```suggestion
supported by this script is the following:
```

File: README.md [hunk 6]
```diff
@@ -1 +1,22 @@
...
+# here you can supply multiple countries if needed
+./scripts/create-country-app.js [...country]
+```
+
+You must supply the country [ISO-3 code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) in lowercase,
+which will be used to create the app name and the folder inside `./packages`.
+
+After running the script, you must then fill the configuration for that country inside
+`./packages/[country-code]/vtex.country-data-settings/configuration.json`. You can take a look
+at how the other countries filled this configuration and go from there.
```
Comment (marcoskwkm) [inline README.md]:
```suggestion
at how this was done for other countries and go from there.
```

**Review by marcoskwkm — APPROVED** [review]
LGTM, just nitpicking on the readme.

**Review by wdsrocha — APPROVED** [review]

**Pull request merged**